models/brac/contact_views/contactview_chp.sql

```diff
@@ -1,6 +1,6 @@
 {{
     config(
-        materialized = 'incremental',
+        materialized = 'view',
```

Comment:
Is changing this to a view intentional?

Reply:
I noticed this model depended on the contactview_chw."@timestamp" column, but the contactview_chw model doesn't have a @timestamp column. Also, contactview_chw and the other models this depends on are created as views. I don't know how to define this as incremental if we don't have a timestamp to calculate. What do you think?

Reply:
I didn't find the field `reported` on `contactview_chw,` only on `contactview_metadata,` so I added it and also to `contactview_chp.` Do you think this is ok?

Reply:
That is fine @lorerod.